tools/docker/Dockerfile

```diff
@@ -177,6 +177,7 @@ COPY --from=firecracker-vm-root-builder /output/vm.ext4 /var/lib/firecracker-con
 COPY --from=firecracker-containerd-build /output/* /usr/local/bin/
 COPY _submodules/runc/runc /usr/local/bin
 COPY tools/docker/firecracker-runtime.json /etc/containerd/firecracker-runtime.json
+COPY tools/docker/scripts/* /tmp/scripts/
```

Comment:
nit: I think I'd prefer something like `/var/firecracker-containerd-test/scripts` over `/tmp/scripts`. It would just be a little more resilient to future changes as `/tmp` is somewhat common to override with tmpfs mounts or bind mounts from the host.

If there's something I'm not thinking of that makes that difficult, no worries, it's a minor concern.

Reply:
No strong preferences on that. Will use /var in that case.

Reply:
Or /usr/local/bin?

Reply:
Sure, /usr/local/bin/ is good too

Reply:
Second thought. I think /var/firecracker-containerd-test/scripts is better. Then we can bind-mount the directory as is, which allows us to remove the hack below.

https://github.com/firecracker-microvm/firecracker-containerd/pull/233/files#diff-5e5cc717e2803524fb4d62e8faea3689R496

Reply:
I might be misunderstanding, but we won't be able to bind mount from the host to inside the microvm. If you create a bind mount for the container it will be a bind mount from the microvm's rootfs to inside the container, not from the actual host outside the microvm to inside the container.

Reply:
Right. I ended-up adding them on Firecracker's host image first, and bind-mount the directory on the guest.